src/test/scala/com/nec/native/TranspileMapUnitSpec.scala

```diff
@@ -144,6 +144,45 @@ final class TranspileMapUnitSpec extends CppTranspilerSpec {
       assertCodeEqualish(genCode1, output1)
     }
 
+    "correctly transpile an If Only Statement" in {
+      val output1 =
+        """
+          | size_t len = in_1[0]->count;
+          |
+          | out_0[0] = nullable_int_vector::allocate();
+          | out_0[0]->resize(len);
+          |
+          | int32_t in_1_val {};
+          |
+          | for (auto i = 0; i < len; i++) {
+          |   in_1_val = in_1[0]->data[i];
+          |
+          |   int32_t x = -1;
+          |
+          |   if ((x < 10)) {
+          |     ({
+          |       x = (x * 3);
+          |       in_1_val;
```

Comment:
While this does indeed look like the Scala code, I'm wondering if it is compilable C++ code.

Reply:
It should be compilable with `-std=gnu++17`, though the compiler will warn about an unused value.